### ISSUE TITLE
tab panel警告问题

### ISSUE THREAD

**Issue by anncwb** [vueComponent/ant-design-vue#2877]
- [ ] I have searched the [issues](https://github.com/vueComponent/ant-design-vue/issues)         of this repository and believe that this is not a duplicate.
        
        
### Version
2.0.0-beta.5

### Environment
vue3 vite

### Reproduction link
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/embed/2wpk21kzvr)

### Steps to reproduce
在jsx使用setup返回tab.panel时候报警告 setup() return property "_isActived" should not start with "$" or "_" which are reserved prefixes for Vue internals.  注释tab.panel组件消失

### What is expected?
删除警告

### What is actually happening?
出现警告

**Comment by anncwb** [comment]
```
setup() return property "_isActived" should not start with "$" or "_" which are reserved prefixes for Vue internals
```

**Comment by John60676** [comment]
This problem has been fixed  #2865
